src/csv_to_json.py

```diff
@@ -0,0 +1,50 @@
+"""
+CSVファイルをJSONファイルに変換するクラス.
+
+@author Keiya121
+"""
+
+import csv
+import json
+import os
+from typing import List
+
+
+class CSVToJSONConverter:
+    """CSVファイルをJSONファイルに変換するクラス."""
+
+    def __init__(self, csv_file_path: str) -> None:
+        """コンストラクタ."""
```

Comment:
```suggestion
        """コンストラクタ."""

        Args:
            csv_file_path (str): CSVファイルのパス
        """
```
こんな感じで、他の箇所も含めて、docstringに Args などを追記してください。